modules/install-sno-additional-requirements-for-installing-sno-on-a-cloud-provider.adoc

```diff
@@ -0,0 +1,18 @@
+// This module is included in the following assemblies: 
+//
+// installing/installing_sno/install-sno-preparing-to-install-sno.adoc
+
+:_content-type: CONCEPT
+[id="additional-requirements-for-installing-sno-on-a-cloud-provider_{context}"]
+= Additional requirements for installing {sno} on a cloud provider
+
+The documentation for installer-provisioned installation on cloud providers is written with a high availability cluster consisting of three control plane nodes. When referring to the documentation, consider the differences between the requirements for a {sno} cluster and a high availability cluster.
```

Comment:
>The documentation for installer-provisioned installation on cloud providers is written with a high availability cluster consisting of three control plane nodes

This sentence feels incomplete. Should "in mind" follow? Or maybe "with" should be replaced with "for"?